### ISSUE TITLE
Error: Attempt to change mapping for "form" extension from "application/x-www-form-urlencoded" to "application/x-www-form-urlencoded". Pass `force=true` 

### ISSUE THREAD

**Issue by lintdeveloper** [AfricasTalkingLtd/africastalking-node.js#94]
I have issues when building app in production using africas talking api

```
e.js:56
        throw new Error(
              ^
Error: Attempt to change mapping for "form" extension from "application/x-www-form-urlencoded" to "application/x-www-form-urlencoded". Pass `force=true` to allow this, otherwise remove "form" from the list of extensions for "application/x-www-form-urlencoded".
    at Mime.define (/Users/mussa/Documents/Source/Vircap/vircap-backend/node_modules/mime/Mime.js:56:15)
    at Object.<anonymous> (/Users/mussa/Documents/Source/Vircap/vircap-backend/node_modules/unirest/index.js:26:6)
```

**Comment by Tyak99** [comment]
I'm having this same issue and I believe it's related to this open PR in unirest [https://github.com/Kong/unirest-nodejs/pull/129](https://github.com/Kong/unirest-nodejs/pull/129)
Please is there a possible fix aside hoping the unirest contributors merge the dependency fix?

**Comment by DaggieBlanqx** [comment]
Hello @lintdeveloper and @Tyak99, are you still experiencing this error?
If yes, could you help me reproduce it so we can look into it and fix it?

**Comment by DaggieBlanqx** [comment]
Issue update:
In commt [fc7a59a](https://github.com/AfricasTalkingLtd/africastalking-node.js/pull/101/commits/699616b9d37c8c56a19ac64d57efbc4117da580e), we dropped unirest for axios.
The error that you experienced was traced back to unirest.
Thus, by removing unirest, this issue too was resolved.
Thank you @lintdeveloper and @Tyak99 for bringing this error to our attention and for helping us improve this SDK for everyone.
